textile/fabric_printing/doctype/coating_order/coating_order.py

```diff
@@ -0,0 +1,200 @@
+# Copyright (c) 2023, ParaLogic and contributors
+# For license information, please see license.txt
+
+import frappe
+from frappe import _
+from frappe.utils import flt
+from erpnext.stock.get_item_details import is_item_uom_convertible
+from textile.controllers.textile_order import TextileOrder
+from textile.fabric_printing.doctype.print_order.print_order import get_fabric_item_details
+from textile.utils import get_textile_conversion_factors, validate_textile_item
+
+import copy
+
+
+force_fields = ["customer_name", "fabric_item_name", "fabric_material",
+				"fabric_type", "fabric_width", "fabric_gsm", "fabric_per_pickup"]
+
+
+class CoatingOrder(TextileOrder):
+	def onload(self):
+		self.set_fabric_stock_qty()
+
+	def validate(self):
+		self.set_missing_values()
+		self.validate_dates()
+		self.validate_customer()
+		self.validate_fabric_item()
```

Comment:
use `TextileOrder.validate_fabric_item(textile_item_type)` instead of overriding